lib/standalone.js

```diff
@@ -246,7 +249,7 @@ async function standalone({
 		});
 
 		stylelintResults = await Promise.all(getStylelintResults);
-	} else if (allowEmptyInput) {
+	} else if (allowEmptyInput || config?.allowEmptyInput) {
```

Comment:
[suggest] I have the same suggestion as above:

```suggestion
	} else if (allowEmptyInput ?? config?.allowEmptyInput) {
```